lkmpg.tex

```diff
@@ -1415,6 +1415,8 @@ \section{System Calls}
 But what if you want to do something unusual, to change the behavior of the system in some way?
 Then, you are mostly on your own.
 
+Notice that this example has been unavailable since v6.9, specifically after this \href{https://github.com/torvalds/linux/commit/1e3ad78334a69b36e107232e337f9d693dcc9df2#diff-4a16bf89a09b4f49669a30d54540f0b936ea0224dc6ee9edfa7700deb16c3e11R52}{commit}, due to the system call table changing the implementation from an indirect function call table to a switch statement.
```

Comment:
1. Use "Linux v6.9" instead of plain "v6.9" to avoid confusion.
2. Split the part starting with "specifically after" into a newline for consistency.
3. Explain the considerations behind indirect function calls.